### PR TITLE
(DOCSP-25433) Remove 3.6 note & tweak db triggers

### PR DESCRIPTION
**Pull request by nlarew**
## Pull Request Info

See also https://github.com/10gen/cloud-docs/pull/3713

### Jira

- (DOCSP-25433) Remove 3.6 note & tweak db triggers

### Staged Changes (Requires MongoDB Corp SSO)

- [Database Triggers](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/triggers/triggers/database-triggers/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/triggers/database-triggers.txt [hunk 1]
```diff
@@ -16,41 +16,46 @@ Overview
...
-
-You can use a :manual:`$match </reference/operator/aggregation/match>`
-expression document to filter change events based on
-field values. For more information, see :manual:`Query on Embedded
-Documents </tutorial/query-embedded-documents>` in the MongoDB server
-manual.
+document is added, updated, or removed in a linked {+atlas+} cluster.
+Unlike SQL data triggers, which run on the database server, triggers run
+on a serverless compute layer that scales independently of the database
+server.
```
Comment (mongodben) [inline source/triggers/database-triggers.txt:22]: somewhere in this 1st paragraph, you should mention that Triggers invoke **Atlas Functions** (and perhaps alternatively AWS Event Bridge)

**Review by mongodben — COMMENTED** [review]
generally looks good. one comment to address.

will review the cloud-docs PR after changes are incorporated here.

File: source/triggers/database-triggers.txt (hunk 1 above)
Comment (nlarew) [reply source/triggers/database-triggers.txt]: Done!

File: source/triggers/database-triggers.txt [hunk 2]
```diff
@@ -16,41 +16,47 @@ Overview
...
-You can use a :manual:`$match </reference/operator/aggregation/match>`
-expression document to filter change events based on
-field values. For more information, see :manual:`Query on Embedded
-Documents </tutorial/query-embedded-documents>` in the MongoDB server
-manual.
+document is added, updated, or removed in a linked {+atlas+} cluster.
+Unlike SQL data triggers, which run on the database server, triggers run
+on a serverless compute layer that scales independently of the database
+server. Triggers automatically call :ref:`Atlas Functions <functions>`
+and can forward events to external handlers through AWS EventBridge.
```
Comment (mongodben) [inline source/triggers/database-triggers.txt:23]: prob worth linking to the page https://www.mongodb.com/docs/atlas/app-services/triggers/examples/send-events-aws-eventbridge/ or the EventBridge docs here

**Pull request merged**